src/claimList.js

```diff
@@ -14,38 +25,223 @@ if (document.readyState === "loading") {
 }
 
 function executeOverride() {
-  const allClaims = getMetadata();
-  updateIcon();
   setupAnalytics();
-  logView(allClaims);
+  makeMobileFriendly();
+
+  const { claims, name } = getMetadata();
+  removeOldHtml();
+  addNewHtml(name, claims);
+
+  styleBody();
   addFeedbackLink();
+  updateIcon();
+  logView(claims);
 }
 
 function getMetadata() {
-  return Array.from(
-    document.getElementsByTagName("table")[2]?.children[0]?.children
-  )
+  const tableElements = document.getElementsByTagName("table");
+  const claims = Array.from(tableElements[2]?.children[0]?.children)
     .slice(1)
     .map((row, i) => ({
-      claimType: row.children[1].innerText.trim(),
-      claimDate: row.children[2].innerText.trim(),
-      claimStatus: row.children[3].innerText.trim(),
+      seqNum: row.children[0].innerText.trim(),
+      type: row.children[1].innerText.trim(),
+      date: row.children[2].innerText.trim(),
+      status: row.children[3].innerText.trim(),
     }));
+  const name =
+    tableElements[1]?.children[0]?.children[0]?.children[0]?.innerText;
+  return { claims, name };
+}
+
+function removeOldHtml() {
+  const root = document.getElementsByName("claimlist")[0]?.children;
+  root[3]?.remove();
```

Comment:
Dust: wondering if its worth just checking that root is defined and has a minimum length before all the remove methods

Reply:
Ah I see. I had just done `root[3]?.remove()` so that I only call the method if those children are defined. I think it's a fair point to explicitly check for the right number of children though, because if it's not the case, then something is clearly wrong, and we probably shouldn't remove just a subset